### ISSUE TITLE
Strange Warnings in TestRepartitionAndDistribution2D

### ISSUE THREAD

**Issue by fsimonis** [precice/precice#459]
## Problem
The test `TestRepartitionAndDistribution2D` emits strange warnings regarding the count of filtered meshes: `1 of 0 vertices` looks fishy.

## Warning
```
(1) WARNING: 2 of 6 vertices of mesh SolidzMesh have been filtered out since they have no influence on the mapping.
(0) WARNING: 1 of 0 vertices of mesh MyMesh have been filtered out since they have no influence on the mapping.
```
## Location
https://github.com/precice/precice/blob/4ab406fb065196f2b8d701705ae67daf8d6a4334/src/partition/ReceivedPartition.cpp#L475-L479

**Comment by uekerman** [comment]
I think the warning message is wrong. 
`_mesh->vertices().size()` is the size of the mesh after filtering on the master rank. 
`_mesh->getGlobalNumberOfVertices` should be what we want.

**Comment by uekerman** [comment]
Also, Neda's Fence gives
```
38400 of 7744 vertices of mesh Fluid_Surface_lineuler_Write have been filtered out since they have no influence on the mapping.
```

**Comment by atotoun** [comment]
I agree with Benjamin:
1- ``_mesh->vertices().size()`` is the number of vertices for received mesh in master rank after filtering
2- in this test, master rank has no vertices in the interface, so the above number is zero
3- ``_mesh->getGlobalNumberOfVertices`` is the total number of received mush. This is what we want here.

**Comment by uekerman** [comment]
fixed in #501
